examples/camera-app/linux/src/clusters/camera-avsettingsuserlevel-mgmt/camera-avsettingsuserlevel-manager.cpp

```diff
@@ -232,11 +233,13 @@ Status CameraAVSettingsUserLevelManager::DPTZRelativeMove(uint16_t aVideoStreamI
                 {
                     // if the delta would move us out of the cartesian plane of the sensor, limit to the top hand edge
                     //
-                    uint16_t y1Movement = ((deltaY < 0) && (abs(deltaY) > viewport.y1)) ? -viewport.y1 : deltaY;
-                    viewport.y1         = static_cast<uint16_t>(viewport.y1 + y1Movement);
+                    uint16_t y1Movement =
```

Comment:
why are these two unsigned when the others are signed?

I think this can be negative if it ends up being -viewport.y1

Reply:
Good catch! I think I just copied & pasted wrongly. Updated.